google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/StorageObject.scala

```diff
@@ -104,15 +104,15 @@ final class StorageObject private (
   def withTimeCreated(value: OffsetDateTime): StorageObject = copy(timeCreated = value)
   def withTimeDeleted(value: OffsetDateTime): StorageObject = copy(timeDeleted = Some(value))
   def withStorageClass(value: String): StorageObject = copy(storageClass = value)
-  def withContentDisposition(value: String): StorageObject = copy(contentDisposition = value)
-  def withContentEncoding(value: String): StorageObject = copy(contentEncoding = value)
-  def withContentLanguage(value: String): StorageObject = copy(contentLanguage = value)
+  def withContentDisposition(value: String): StorageObject = copy(contentDisposition = Some(value))
```

Comment:
Should this not be Option(_) instead of (Some) to accept null?

Reply:
@ennru do you have some guidance here on what I should do for the java api?

Reply:
Good point.  Please update any optional reference types to use `Option.apply`.

Reply:
Let's keep the optional case class fields as scala `Option` so the spray serdes works correctly.  Please add Java getters that return `Optional` fields (i.e. see `getContentType`).  Here's a simple one liner to wrap an `Option`: `Optional.ofNullable(option.orNull)`.

Reply:
that line does not work with pimitives, used scala.compat.java8.OptionConverters